ClassicThemePlugin.inc.php

```diff
@@ -209,4 +219,15 @@ public function hasAuthorsInfo($hookName, $args) {
 		$templateMgr->assign('boolAuthorInfo', $boolAuthorInfo);
 	}
 
+	public function homepageJournalSummary($hookName, $args) {
+		$templateMgr = $args[0];
+		$template = $args[1];
+
+		if ($template != "frontend/pages/indexJournal.tpl") return false;
+
+		$templateMgr->assign(array(
+			'showJournalSummary' => $this->getOption('journalSummary'),
+		));
+	}
```

Comment:
@Vitaliy-1, @sssoz and @thinkbulecount2: just letting you know that from v3.2, all templates will have access to a `{$activeTheme}` variable. So in your theme's template files you can do:

```
{if $activeTheme->getOption('journalSummary')}
```

That should help reduce the amount of helper hooks you need to use to get option data into the templates. :+1:

Reply:
I will create an issue in all themes to update that once implemented.